include/fmt/color.h

```diff
@@ -409,16 +413,19 @@ template <typename Char> struct ansi_color_escape {
     buffer[19] = static_cast<Char>(0);
   }
   FMT_CONSTEXPR ansi_color_escape(emphasis em) FMT_NOEXCEPT {
-    uint8_t em_codes[4] = {};
+    uint8_t em_codes[8] = {};
```

Comment:
Let's add a constant for the number of emphases (`num_emphases`) and use it here and below.